feature-account-api/src/main/java/io/novafoundation/nova/feature_account_api/data/model/Fee.kt

```diff
@@ -14,25 +18,31 @@ sealed interface Fee {
      * Information about origin that is supposed to send the transaction fee was calculated against
      */
     val submissionOrigin: SubmissionOrigin
+
+    val paymentAsset: PaymentAsset
```

Comment:
There is no point on distringuishing on data level between `Native` and `Asset` cases in a form of sealed class
`Fee`  should only be concerned about asset itself
Everything else is unnecessary complexity

Reply:
Okay. Fixed it.